llabs is defined in C99. features.h on other platforms enables C99 features
automatically, here we explicitly enable them with _GNU_SOURCE (which enables
also a lot of other things).
--- a/src/libstrongswan/utils.c
+++ b/src/libstrongswan/utils.c
@@ -14,6 +14,7 @@
  * for more details.
  */
 
+#define _GNU_SOURCE
 #include "utils.h"
 
 #include <sys/stat.h>
